components/Player.js

```diff
@@ -0,0 +1,60 @@
+import propTypes from 'prop-types'
+import Card from './Card'
+
+/**
+ * Player list component
+ */
+export default function Player(props) {
+  return (
+    <div className="flex md:flex-col rounded border border-slate-300 my-2">
+      <div className=" flex w-full justify-between">
+        <div className="flex ">
+          <div
+            className={`flex items-center justify-center rounded-lg p-2  ${props.bgColor}`}
+          >
+            <span className="   bg-slate-300 rounded-full h-14 w-14 ">
+              {props.letter}
+            </span>
+          </div>
+
+          <p className="px-2 my-auto text-body font-body font-semibold text-slate-700">
+            {props.playerName}
+          </p>
+        </div>
+
+        {/* <p className="px-6 my-auto">{props.playerCard}</p> */}
+
+        {props.selectedCard ? (
+          <Card
+            src={props.selectedCard.src}
+            id={props.selectedCard.id}
+            key={props.selectedCard.id}
+            alt={props.selectedCard.alt}
+            className=" "
+          />
+        ) : (
+          'other peoples cards here'
```

Comment:
Could we use a default value for when no cards are selected such as this? Might need to use states somehow.
![image](https://user-images.githubusercontent.com/53541292/157697333-03fbe1ff-c84b-4ab3-9415-3a547fd90827.png)